tests/api/v1/collectors/heartbeat.js

```diff
@@ -201,31 +200,25 @@ describe('tests/api/v1/collectors/heartbeat.js >', () => {
           api.post(`/v1/collectors/${collector1.id}/heartbeat`)
           .set('Authorization', collectorTokens[collector1.name])
           .send({ timestamp: Date.now() })
-          .expect(403)
+          .expect(200)
```

Comment:
prefer http status constants over the numbers here

Reply:
ok will change